eslint.config.mjs

```diff
@@ -575,4 +576,36 @@ export default tseslint.config(
       'import/no-default-export': 'off',
     },
   },
+  {
+    extends: [perfectionistPlugin.configs['recommended-alphabetical']],
+    files: ['packages/utils/src/**/*.ts'],
+    rules: {
+      'perfectionist/sort-classes': [
+        'error',
+        {
+          order: 'asc',
+          partitionByComment: true,
+          type: 'natural',
+        },
+      ],
+      'perfectionist/sort-objects': [
+        'error',
+        {
+          order: 'asc',
+          partitionByComment: true,
+          type: 'natural',
+        },
+      ],
+      'perfectionist/sort-union-types': [
+        'error',
+        {
+          order: 'asc',
+          groups: ['unknown', 'nullish'],
+          ignoreCase: false,
```

Comment:
Shall we revert this now that the `| null | undefined` issue is resolved?

Reply:
Yeah, I'd been using it to put lowercase keywords like `string` after capitalized things... `groups: ['unknown', 'keyword', 'nullish']` does that too and a little better.